qualification/antenna_channelised_voltage/test_delay.py

```diff
@@ -188,6 +188,9 @@ async def set_delays(delays: list[str]) -> None:
 
 
 @pytest.mark.requirements("CBF-REQ-0187,CBF-REQ-0188")
+@pytest.mark.xfail(
+    reason="requirement cannot be met for all modes as delays cannot be updated more than once per spectrum"
+)
```

Comment:
Does x-fail cause problems if some subset of the tests pass?

Reply:
Those are pass are reported as "xpass" when pytest summarises how many tests did what, but that counts as a pass. You can make xpass a failure by specifying `strict=True` on the mark.

I would have preferred all the xfails to be conditional on the specific modes that we know fail, but I don't see a simple way to give it a filter that has access to the test parameters.